src/System.Net.Security/tests/System.Net.Security.Tests.builds

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="FunctionalTests\System.Net.Security.Tests.csproj">
-      <TestTFMs>netcoreapp1.0;net46</TestTFMs>
+      <TestTFMs>netcoreapp1.1;net463</TestTFMs>
```

Comment:
@joperezr @weshaggard Isn't this automatically tested somewhere? If not we need to make sure we have a tracking test issue.